maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -287,6 +301,29 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun fetchSession(
+        request: MaestroAndroid.SessionRequest,
+        responseObserver: StreamObserver<MaestroAndroid.SessionResponse>
+    ) {
+        val sessionReq = Request.Builder()
+            .get()
+            .url("http://localhost:7008/session")
```

Comment:
This service is running in the same process, what is the point of using HTTP to communicate with it? You can just access memory directly

Reply:
Specifically, you are looking for `SdkService` class